static/app/views/performance/browser/webVitals/utils/queries/useInpSpanSamplesWebVitalsQuery.tsx

```diff
@@ -30,6 +30,8 @@ export function useInpSpanSamplesWebVitalsQuery({
   const {data, isLoading, ...rest} = useSpansIndexed(
     {
       search: MutableSearch.fromQueryObject({
+        has: 'message',
```

Comment:
For some reason, `has:span.description` doesn't seem to work (probably due to some special case handling in discover). `message` does work though, and it's mapped to the same tag in discover, so we're using that instead.